docs/compiler/cli/testing.md

```diff
@@ -0,0 +1,45 @@
+# Testing of your application
+
+Testing of your qooxdoo app can be done by the qooxdoo cli command `qx test`.
+For this purpose you need to install one of the provide test plugins:
+  - qooxdoo/qxl.testtapper: Runs units tests based on `qx.dev.unit.TestCase`
+
+## Preparing your application
+
+  - install testapper with `qx package install qooxdoo/qxl.testtapper --save=0`

```

Comment:
Should this also specify to install `qooxdoo/qxl.logpane` as it seemed I needed to?

Reply:
no should be done with Installation of testtapper. The requires entry was missing.